### ISSUE TITLE
Remove all deprecated things

### ISSUE THREAD

**Issue by PowerKiKi** [PHPOffice/PhpSpreadsheet#22]
Deprecated things should either be dropped entirely and their replacement be documented, or the `@deprecated` mentions should be dropped. As noted in https://github.com/PHPOffice/PHPExcel/issues/1029 it's unfortunately not always clear why it was deprecated in the first place.

**Comment by Quix0r** [comment]
This might be a clear way of deprecation:

1. mark the method or code block as `@deprecated` and document a replacement
2. after x _time-unit_ (e.g. 1/2 year?) remove the deprecated code

**Comment by PowerKiKi** [comment]
This issue is specifically for already deprecated things that need to be dropped before the first release. We are beyond the deprecating phase (by several years in some cases). While I agree with you for future deprecations, I don't think it applies here.

**Comment by PowerKiKi** [comment]
@MarkBaker I worked on removing deprecated methods and [documenting their replacements](http://phpspreadsheet.readthedocs.io/en/develop/topics/migration-from-PHPExcel/#removed-deprecated-things).

But I got stuck on `getCalculatedValue()`, `setFormulaAttributes()` and `setFormulaAttributes()`, because I couldn't find any replacement. They've been deprecated in 59932b0cac4aec4ba13b17f9f36b0b939bc083c4 and the changelog mention that replacements will be added in the future, but it seems that it's never been done. It seems that everything is related to https://github.com/PHPOffice/PHPExcel/issues/9. Is it still something that you are working on ? What should we do about `getCalculatedValue()` ?
